main.go

```diff
@@ -171,7 +171,14 @@ func notify(host, desc string, cfg *ini.File, days int, err error, verbose bool)
 		log.Printf("notify: sending host %s expiration notification to %s", host, section.Key("rcpt").String())
 	}
 
-	if err := smtp.SendMail(mailhost+":"+port, auth, section.Key("from").String(), to, msg); err != nil {
+	errc := make(chan error, 1)
+	go func() {
+		errc <- smtp.SendMail(mailhost+":"+port, auth, section.Key("from").String(), to, msg)
+	}()
+	select {
+	case err := <-errc:
 		log.Fatalf("could not send email: %s", err)
+	case <-time.Tick(30 * time.Second):
```

Comment:
It doesn't really matter because the program is about to terminate anyway, but [time.Tick leaks a goroutine](https://godoc.org/time#Tick). You can use [time.After](https://godoc.org/time#After) instead to avoid the leak.

Reply:
fixed!